extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -116,6 +119,17 @@
   height: var(--thumb-size) !important;
   background: var(--interactive-accent-color) !important;
   border-radius: 50% !important;
+  position: relative;
+}
+
+.i-amphtml-story-interactive-slider-container.show
+  .i-amphtml-story-interactive-slider-input::-webkit-slider-thumb {
+  -webkit-appearance: none !important;
+  transform: rotate(45deg) !important;
+  box-shadow: 0em 0em 0.5em rgba(0, 0, 0, 0.25) !important;
+  z-index: 3 !important;
```

Comment:
You can move the `z-index: 3` to the rule above so the element doesn't change the `z-index` between states

Reply:
Moving the z-index up, causes the thumb to cover the post state bubble. However, I still need the z-index in this class because it keeps the thumb infront of the bubble during the mid state.